src/components/ActionButton.tsx

```diff
@@ -107,6 +110,8 @@ export default function ActionButton({
         return 'onAccent'
       case 'accentSoft':
         return 'accent'
+      case 'activeSoft':
+        return 'active'
```

Comment:
i think we don't need this anymore right

Reply:
ah yeah i think it's unused. thanks